src/theme.js

```diff
@@ -95,48 +63,36 @@ const theme = {
 		'16rem', // 256px
 	],
 	fonts: {
-		body: 'Helvetica Neue, Helvetica, Arial, sans-serif',
 		heading: 'Helvetica Neue, Helvetica, Arial, sans-serif',
-		monospace: 'Menlo, monospace',
+		body: 'Helvetica Neue, Helvetica, Arial, sans-serif',
+		monospace: `"Mono Lisa", Monaco, monospace` /* Check out https://monolisa.dev for Mono Lisa font */,
 	},
 	fontSizes: {
-		xl: '4em',
-		lg: '2em',
-		md: '1em',
-		sm: '0.9em',
-		xs: '0.75em',
+		xl: '4rem',
+		lg: '2rem',
+		md: '1rem',
+		sm: '0.9rem',
+		xs: '0.75rem',
 	},
 	fontWeights: {
 		light: 200,
 		normal: 400,
 		bold: 700,
 	},
 	lineHeights: {
-		base: 1.5,
+		body: 1.5,
 		heading: 1.1,
 	},
-	textStyles,
-	colors,
 	borders: {
 		none: 'none',
 		thin: '1px solid',
 	},
 	radii: {
-		base: '0.15em',
-	},
-	shadows: {
-		focus: `0 0 0 2px ${colors.focus}`,
-	},
-};
-
-export const inverted = {
-	...theme,
-	colors: {
-		...getColors(invertedPalette),
-		primary: invertedPalette.grey[4],
-		hover: invertedPalette.grey[6],
-		focus: invertedPalette.grey[1],
+		none: 0,
+		base: '0.25em',
+		round: 99999,
```

Comment:
I think this is more common?

```suggestion
		round: '50%',
```

Reply:
Doesn't work as expected though. It will create ovals.

Reply:
Yeah, I guess the initial shape should be square.

Reply:
Yep, that's why I'm using `99999` these days.